internal/fleet/enrollment_tokens_data_source.go

```diff
@@ -0,0 +1,132 @@
+package fleet
+
+import (
+	"context"
+
+	"github.com/elastic/terraform-provider-elasticstack/internal/clients/fleet"
+	"github.com/elastic/terraform-provider-elasticstack/internal/utils"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+func DataSourceEnrollmentTokens() *schema.Resource {
```

Comment:
TIL Enrollment is US English, aka incorrect.

Reply:
Just to be clear, I'm not suggesting we change it. The ES resource is an enrollment token, I've just never read the word that closely before to notice it's poorly spelled.

Reply:
Also, I'm joking. It's just that, well English is in the name so maybe the version used in England is arguably more correct. 

Maybe we should just discuss it over a nice biscuit whilst relaxing in our thongs.

Reply:
> The documentation uses the US English variant, so I ended up going with that spelling.

:100: Internal consistency >>>> My personal opinion on the matter.